### DISCUSSION TITLE
Support for App to App authentication

### DISCUSSION THREAD

**Discussion by jcubic** [community/community#51846] [Mobile]
### Select Topic Area

Question

### Body

When you install the GitHub app it hijacks all the links even from apps, and you can't authenticate using OAuth. If you registered on a website using GitHub and then want to use a mobile app for the same service.

I had this problem with Element (Matrix client) when connecting to gitter.im chat. It tries to log in but fails because the mobile app expects the same behavior as the browser. The GitHub app fails to provide such a function.

This was reported to GitHub support and they suggested also adding this to the GH community.

**EDIT**: I've added this as default question and there is no way to change that now. It should be some kind of discussion.

**Comment by iskae** [comment]
Hello @jcubic 👋 

Thanks so much for reaching out! We have tried to reproduce the issue quite a bit, but unfortunately were not able to reproduce it 😅 

Would you be able to provide us with a recording of the issue so that we can pinpoint exactly what is happening?

Cheers!

**Reply by jcubic** [reply]
You can find video on GitLab the same was sent to GitHub support

https://gitlab.com/gitterHQ/gitter.im/-/issues/8#note_1340393868

**Reply by jcubic** [reply]
I was trying to upload the video but there is a limit of 10MB.

**Reply by iskae** [reply]
That is totally ok 😊 Thanks so much for the video 🙇 We are investigating why we are still catching the intent even after browser has been selected for handling 👍 

We will try to fix it as soon as possible 😊 

Please stay tuned for the future releases!

Cheers!

**Reply by jcubic** [reply]
Awesome thank. I was expecting that this will not e fixed after contacting GH support.

**Reply by maximepvrt** [reply]
same error here https://github.com/nuxt/nuxters/issues/22 (and Brave Browser is not listed on [Disambiguation Dialog](https://developer.android.com/training/basics/intents/sending#disambiguation-dialog))

Can you exclude catch https://github.com/login/oauth/authorize url on github mobile app ?

**Comment by maximepvrt** [comment]
same error here https://github.com/nuxt/nuxters/issues/22 (and Brave Browser is not listed on [Disambiguation Dialog](https://developer.android.com/training/basics/intents/sending#disambiguation-dialog))

Can you exclude catch https://github.com/login/oauth/authorize url on github mobile app ?
